src/main/java/nextstep/mission/Line.java

```diff
@@ -0,0 +1,74 @@
+package nextstep.mission;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Line {
+    private List<Boolean> points;
+
+    public Line(List<Boolean> points) {
+        validate(points);
+
+        this.points = points;
+    }
+
+    public Line(int size) {
+        List<Boolean> points = createPoints(size);
+
+        validate(points);
+
+        this.points = points;
+    }
```

Comment:
반복되는 생성 로직은 통합시켜주는게 좋아보입니다:)
```suggestion
    public Line(List<Boolean> points) {
        validate(points);
        this.points = points;
    }

    public Line(int size) {
        this(createPoints(size));
    }
```

Reply:
오 이건 몰랐던 방식이네요. 감사합니다!!